modules/node/persistence/cms_node_storage_extension.e

```diff
@@ -49,6 +49,13 @@ feature -- Persistence
 			end
 		end
 
+	delete_by_id (a_nid: INTEGER_64)
```

Comment:
I think we should have

delete (a_node: CMS_NODE) 
   ...

especially when callers all have access to the node object.

An advantage of this, is to have access to the full object .. and if ever one day, we change the type of nodeid ... we won't have to change many signatures ;)